spec/paths/coupons-redemptions@{id}@cancel.yaml

```
@@ -4,6 +4,7 @@ post:
   tags:
     - Coupons
   summary: Cancel a coupon redemption
+  operationId: PostCouponRedemptionCancellationRequest
```

```suggestion
  operationId: PostCouponRedemptionCancellationRequest
```
The `Request` suffix is not necesary here.

---

Ok ,wo what is the rule, where the suffix is necessary and where it is not?

---

I just post my thoughts https://github.com/Rebilly/RebillyAPI/pull/728#pullrequestreview-267660850